contracts/Refunder.sol

```diff
@@ -88,7 +88,7 @@ contract Refunder is ReentrancyGuard, OwnableUpgradeable, PausableUpgradeable, I
         address targetContract,
         bytes4 interfaceId,
         bool isRefundable_
-    ) external override onlyOwner nonReentrant {
+    ) external override onlyOwner nonReentrant whenNotPaused{
```

Comment:
I don't think that we should have pausable for updating the refundable. We need pausable only for the `relayAndRefund` function